pt-BR/mail/chrome/messenger/preferences/applications.dtd

```diff
@@ -24,14 +24,14 @@ entities below were automatically added by l10nmerge
 -->
 
 <!ENTITY enableCloudFileAccountOffer.mb "MB">
-<!ENTITY authRequired.button.label        "Authorize">
+<!ENTITY authRequired.button.label        "Autorizar">
 <!ENTITY authRequired.button.accesskey    "U">
 <!ENTITY addCloudFileAccount.accesskey    "A">
-<!ENTITY removeCloudFileAccount.label     "Remove">
-<!ENTITY addCloudFileAccount.description  "Add a new online storage service">
-<!ENTITY authRequired.description         "Your authorization is required in order to see the settings for this storage service.">
-<!ENTITY addCloudFileAccount.label        "Add">
+<!ENTITY removeCloudFileAccount.label     "Remover">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY removeCloudFileAccount.label     "Excluir">